datastore/google/cloud/datastore/client.py

```diff
@@ -116,28 +116,29 @@ def _extended_lookup(connection, project, key_pbs,
     while loop_num < _MAX_LOOPS:  # loop against possible deferred.
         loop_num += 1
 
-        results_found, missing_found, deferred_found = connection.lookup(
+        lookup_response = connection.lookup(
             project=project,
             key_pbs=key_pbs,
             eventual=eventual,
             transaction_id=transaction_id,
         )
 
-        results.extend(results_found)
+        # Accumulate the new results.
+        results.extend(result.entity for result in lookup_response.found)
 
         if missing is not None:
-            missing.extend(missing_found)
+            missing.extend(result.entity for result in lookup_response.missing)
```

Comment:
Nevermind, I see it's in the [proto](https://github.com/googleapis/api-client-staging/blob/4ab660906bca3bae2620e1814c58b4368781a735/generated/python/proto-google-cloud-datastore-v1/google/cloud/proto/datastore/v1/datastore_pb2.py#L130) with a default of `[]`.